konfera/models/core.py

```diff
@@ -0,0 +1,79 @@
+EVENT_TYPE_CHOICES = (
+    ('conference', ('Conference')),
```

Comment:
you probably forgot _ and import at the begging of the file as 'Conference' was meant to be translation string.

```
from django.utils.translation import ugettext_lazy as _

('conference', _('Conference')),
```